netcontrold/lib/dataif.py

```diff
@@ -332,6 +334,45 @@ def __cls_repr__(cls):
     return Dataif_Port
 
 
+class Dataif_Coverage(object):
+    """
+    Class to represent the slowpath counters.
```

Comment:
instead of "slowpath", say "coverage" .. as this class abstracts coverage counters.

Reply:
ok